Implement User Interface #83

E0011875 opened the pull request:
Closes #83

Commit:
1) Remove DarkTheme.css, address_book_32.png
2) Modify MainWindow.fxml, PersonListCard.fxml
3) Replace calendar.png, clock.png
4) Added iManager.png and settng.png

Commit:
1) Remove DarkTheme.css, address_book_32.png
2) Modify MainWindow.fxml, PersonListCard.fxml
3) Replace calendar.png, clock.png
4) Added iManager.png and settng.png

Commit:
Modify fxml files.

Commit:
Modify fxml files.

Commit:
1) Fix indentation error
2) Remove mainPane from ResultDisplay.java
3) Added startEndOnLY & deadlineOnly HBox to PersonCard.java
4) Added startDate & endDate to PersonCard.java
5) Change MIN_HEIGHT & MIN_WIDTH in MainWindow.java
6) Change Application Icon
7) Disable Setting button
8) Apply RedTheme.css to UiManager

Commit:
1) Fix indentation error
2) Remove mainPane from ResultDisplay.java
3) Added startEndOnLY & deadlineOnly HBox to PersonCard.java
4) Added startDate & endDate to PersonCard.java
5) Change MIN_HEIGHT & MIN_WIDTH in MainWindow.java
6) Change Application Icon
7) Disable Setting button
8) Apply RedTheme.css to UiManager

Commit:
Modify AppUtilTest to test for iManager_32.png

Commit:
Modify AppUtilTest to test for iManager_32.png

Commit:
Merge branch 'new-ui' of https://github.com/CS2103JAN2017-F14-B3/main into new-ui

LiHaoTan commented on .project:
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Can you revert this file to the one on master for now? I think we need to ask our tutor if we can add this to our .gitignore as different projects have different configuration. Anyway your checkstyle got disabled again.

LiHaoTan commented on src/main/java/seedu/address/ui/PersonCard.java:
```diff
@@ -21,43 +21,43 @@
     private Label id;
     @FXML
-    private Label phone;
+    private HBox startEndOnly;
```
I'm not sure if the variable name ending with `only` is suitable. I think that can be changed in the future though.

LiHaoTan commented on src/main/java/seedu/address/ui/PersonCard.java:
```diff
@@ -21,43 +21,43 @@
+    private HBox startEndOnly;
     @FXML
-    private Label address;
+    private HBox deadlineOnly;
```
Similarly here

LiHaoTan reviewed (changes requested):
Just change the .project file I suppose

LiHaoTan commented on src/main/resources/view/RedTheme.css:
```diff
@@ -0,0 +1,204 @@
+}
+
+.text-field,.text-area, .split-pane,.tab {
+	-fx-border-color: #2e4053;
```
You are mixing tabs and spaces for indentation. The configuration we are given does not auto convert tabs to spaces, but there's no web tools platform on Eclipse for Java Developers so we can't do that either. You can either install it:
http://download.eclipse.org/webtools/updates/

Or just fix the indentation for the few lines since we won't be touching the css file much.

Although we are not told the style to follow for CSS, it's best to keep it consistent to use spaces for CSS as well. Furthermore, mixing spaces and tabs for indentation is not a good idea.

Since you are already using 4 spaces for most of your CSS file, convert all your tabs to spaces

LiHaoTan replied:
You can search for tabs by Ctrl + F , Find: \t, tick regular expressions

LiHaoTan replied:
Anyway if you want a style guide for CSS: https://oss-generic.github.io/process/codingStandards/CodingStandard-Css.html

LiHaoTan reviewed (changes requested):
Fix indentation in .css file as well

Commit:
1) Restore .project from master
2) Fix indentation issue in css files
3) Added Author Tag to RedTheme.css

Commit:
Fix Identation Issue.

Commit:
1) Added author tag to Extension.css
2) Change commandBox colour to yellow on invalid command for better constrast
3) Status-bar colour change to dakr blue for consistency
